spec/models/card_spec.rb

```diff
@@ -4,14 +4,14 @@
   it 'create card with empty original text' do
     card = Card.create(original_text: '', translated_text: 'house', user_id: 1,
                        block_id: 1)
-    expect(card.errors[:original_text]).to include('Необходимо заполнить поле.')
+    expect(card.errors[:original_text]).to include('Заполните поле.')
   end
 
   it 'create card with empty translated text' do
     card = Card.create(original_text: 'дом', translated_text: '', user_id: 1,
                        block_id: 1)
     expect(card.errors[:translated_text]).
-        to include('Необходимо заполнить поле.')
+        to include('Заполните поле.')
```

Comment:
Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.